modules/core/functions.php

```diff
@@ -624,20 +624,98 @@ function check_file_upload($request, $key) {
     return true;
 }}
 
-function privacy_setting_callback($val, $key, $mod) {
-    $setting = Hm_Output_privacy_settings::$settings[$key];
-    $key .= '_setting';
-    $user_setting = $mod->user_config->get($key);
-    $update = $mod->request->post['update'];
+if (!hm_exists('get_nexter_date')) {
+function get_nexter_date($format, $only_label = false) {
```

Comment:
It's always good practice to name functions based on what the developer intends to achieve with their declaration. The current name "get_nexter_date" doesn't convey this effectively.

Reply:
I just renamed this function get_scheduled_date